### PR TITLE
F 625 implement endpoint for donations of campaign

### PR DESCRIPTION
**Pull request by mohammadranjbarz**
related to #625

### CODE REVIEW

File: src/services/campaignTotalDonationsValue/campaignTotalDonationValue.service.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+const { ObjectId } = require('mongoose').Types;
+const { getTotalUsdValueDonatedToCampaign } = require('../../repositories/donationRepository');
+
+module.exports = function verifiedCampaigns() {
```
Comment (aminlatifi) [inline src/services/campaignTotalDonationsValue/campaignTotalDonationValue.service.js:4]: function name should be changed

File: src/services/campaignTotalDonationsValue/campaignTotalDonationValue.service.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  service.docs = {
+    securities: ['get'],
+    operations: {
+      update: false,
+      patch: false,
+      remove: false,
+      create: false,
+      find: {
+        description: 'Get total donation usdValue to a campaign',
+        parameters: [],
```
Comment (aminlatifi) [inline src/services/campaignTotalDonationsValue/campaignTotalDonationValue.service.js:28]:
```suggestion
        parameters: [
          {
            name: 'id',
            description: 'campaign ID',
            in: 'query',
          },
        ],```

**Review by aminlatifi — CHANGES_REQUESTED** [review]

File: src/services/campaignTotalDonationsValue/campaignTotalDonationValue.service.js (hunk 2 above)
Comment (mohammadranjbarz) [reply src/services/campaignTotalDonationsValue/campaignTotalDonationValue.service.js]: @aminlatifi Thanks it's done now

File: src/services/campaignTotalDonationsValue/campaignTotalDonationValue.service.js (hunk 1 above)
Comment (mohammadranjbarz) [reply src/services/campaignTotalDonationsValue/campaignTotalDonationValue.service.js]: Thanks it's done

**Comment by aminlatifi** [comment]
Thanks @mohammadranjbarz

**Pull request merged**